python/tvm/relay/op/contrib/dnnl.py

```diff
@@ -59,15 +59,20 @@ def _func_wrapper(expr):
     return _func_wrapper
 
 
-_register_external_op_helper("nn.batch_norm")
+# _register_external_op_helper("nn.batch_norm")
 _register_external_op_helper("nn.conv2d")
 _register_external_op_helper("nn.dense")
 _register_external_op_helper("nn.relu")
-_register_external_op_helper("add")
-_register_external_op_helper("multiply")
+#_register_external_op_helper("add")
+#_register_external_op_helper("subtract")
+#_register_external_op_helper("multiply")
 
+# _register_external_op_helper("layout_transform")
```

Comment:
Any reason to comment out these op registeration?

Reply:
1. BN op is folded, no need to register it anymore.
2. ADD (dnnl) op only supports the elementwise add, not applicable in the current test models. P.S most of the add ops are fused into conv.
3. Multiply and Subtract are never used.
4. Layout_transform is no need to be registered. (will be removed.)

Reply:
For the recent tested models, perhaps they're useless, but how about the other models? Any drawbacks or concerns to keep them here?

Reply:
BN can be kept. While ADD and Multiply cannot be kept. 
As for densenet121, there exist some isolate ADD ops behind the concat op, they are broadcast add, which is not support by the Binary::Add.